tests/foreman/ui/test_multinetwork.py

```diff
@@ -0,0 +1,588 @@
+        5. Identifier: eth0:0
+        6. Select 'Managed'
+        7. Select 'Virtual Nic'
+        8. attached_to: identifier of primary interface(eth0)
+        9. Go to Infrastructure -> Subnet
+        10. Set IPAM mode to 'Internal DB'
+        11. BootMode 'DHCP'
+        12. Fill all other details correctly in new host form and submit it
+
+        @Assert: Validation error should be raised as you can't configure alias
+        interface in 'DHCP' mode.
+
+        @Status: Manual
+
+        """
+
+    @stubbed()
+    def test_add_alias_interface_5(self):
+        """@Test:  Add an alias interface(eth0:0) when bootMode set to 'Static'
+        mode under selected subnet
+
+        @Feature: Multi Network Support
+
+        @Setup: Provisioning should be configured
+
+        @Steps:
+
+        1. Go to 'Network' tab of 'New host' page
+        2. Click on 'Add Interface' from 'Network tab
+        3. Choose Type: 'Interface'
+        4. mac address: mac should be same as of primary interface
+        5. Identifier: eth0:0
+        5. Select 'Managed'
+        6. Select 'Virtual Nic'
+        7. attached_to: identifier of primary interface(eth0)
+        8. Go to Infrastructure -> Subnet
+        9. Set IPAM mode to 'Internal DB'
+        10. BootMode 'Static'
+        11. Fill all other details correctly in new host form and submit it
+
+        @Assert: Interface should be configured successfully and correct
+        configuration should displayed on proviisoned host
+        under /etc/sysconfig/network-scripts/ifcfg-<interface_name>
+
+        @Status: Manual
+
+        """
+
+    @stubbed()
+    def test_add_alias_interface_6(self):
+        """@Test:  Add an interface with same identifier of exiting interface
+        Like interfaces eth0, eth0:0 already exists and now create new
+        interface with eth0:0 identifier
+
+        @Feature: Multi Network Support
+
+        @Setup: Provisioning should be configured
+
+        @Steps:
+
+        1. Go to 'Network' tab of 'New host' page
+        2. Click on 'Add Interface' from 'Network tab
+        3. Choose Type: 'Interface'
+        4. mac address: mac should be same as of primary interface
+        5. Identifier: eth0:0
+        6. Select 'Managed'
+        7. Select 'Virtual Nic'
+        8. attached_to: identifier of primary interface(eth0)
+        9. Go to Infrastructure -> Subnet
+        10. Set IPAM mode to 'Internal DB'
+        11. BootMode 'Static'
+        12. Fill all other details correctly in new host form and submit it
+
+        @Assert: Validation error should be raised on UI
+
+        @Status: Manual
+
+        """
+
+    @stubbed()
+    def test_delete_alias_interface(self):
+        """@Test:  Delete an alias interface
+
+        @Feature: Multi Network Support
+
+        @Setup: Provisioning should be configured
+
+        @Steps:
+
+        1. Create an aliased interface
+        2. Edit the new host -> Network --> delete the selected interface
```

Comment:
The encoding of this file is declared as UTF-8, so you can use fancy characters like this if you want: →

Not necessary, of course. But it's slick. :)